src/hash/rpo/mod.rs

```diff
@@ -91,6 +91,11 @@ const INV_ALPHA: u64 = 10540996611094048183;
 pub struct Rpo256();
 
 impl Hasher for Rpo256 {
+    /// Rpo256 collision resistance is the same as the security level, that is 128-bits. However,
+    /// out setup of the capacity registers might drop it to 126, so here we use a pessimistic
+    /// approach and pick the lowest possible value.
+    const COLLISION_RESISTANCE: u32 = 126;
```

Comment:
nit: typo "out" should probably be "our".

Also, I'd probably still set it to 128 and leave a `TODO` to address later as this affects other things (e.g., how we define proof security level).